src/extras/primitives/primitives/a-text.js

```diff
@@ -0,0 +1,31 @@
+/* Experimental text primitive.
+ * Issues: color not changing, removeAttribute() not working, mixing primitive with regular entities fails
+ * Color issue relates to: https://github.com/donmccurdy/aframe-extras/blob/master/src/primitives/a-ocean.js#L44
```

Comment:
not sure, but i would assume that means something about problems using a-text primitive with other components in same entity.